MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -29,49 +30,86 @@ class MapboxCoreNavigationTests: XCTestCase {
         
         navigation.locationManager(navigation.locationManager, didUpdateLocations: [depart])
         
-        waitForExpectations(timeout: waitForInterval)
+        waitForExpectations(timeout: waitForInterval) { (error) in
+            XCTAssertNil(error)
+        }
     }
     
     func testLowAlert() {
-        let navigation = RouteController(along: route, directions: directions)
-        navigation.resume()
-        let user = CLLocation(coordinate: CLLocationCoordinate2D(latitude: 37.789118, longitude: -122.432209), altitude: 1, horizontalAccuracy: 1, verticalAccuracy: 1, course: 171, speed: 10, timestamp: Date())
+        let locations = [CLLocation(coordinate: CLLocationCoordinate2D(latitude: 37.789118, longitude: -122.432209),
+                                    altitude: 1, horizontalAccuracy: 1, verticalAccuracy: 1, course: 171, speed: 10, timestamp: Date())]
+        let locationManager = ReplayLocationManager(locations: locations)
+        let navigation = RouteController(along: route, directions: directions, locationManager: locationManager)
         
         self.expectation(forNotification: RouteControllerAlertLevelDidChange.rawValue, object: navigation) { (notification) -> Bool in
             XCTAssertEqual(notification.userInfo?.count, 2)
             
             let routeProgress = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationRouteProgressKey] as? RouteProgress
             let userDistance = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationDistanceToEndOfManeuverKey] as! CLLocationDistance
             
-            return routeProgress?.currentLegProgress.alertUserLevel == .low && routeProgress?.currentLegProgress.stepIndex == 2 && round(userDistance) == 1758
+            return routeProgress?.currentLegProgress.alertUserLevel == .low && routeProgress?.currentLegProgress.stepIndex == 2 && round(userDistance) == 1786
```

Comment:
![screenshot 2017-06-26 22 22 43](https://user-images.githubusercontent.com/764476/27558609-02e9f5f8-5abe-11e7-81ab-f929bc707fad.png)
That test was disabled in the scheme. 😄